tests/net/context/src/main.c

```diff
@@ -68,8 +68,12 @@ static struct k_sem wait_data;
 #define WAIT_TIME K_MSEC(250)
 #define WAIT_TIME_LONG MSEC_PER_SEC
 #define SENDING 93244
-#define MY_PORT 1969
-#define PEER_PORT 16233
+#ifndef CONFIG_NET_CONFIG_MY_PORT
```

Comment:
I think we should not touch the tests, they are constructed like this so that they will pass the given tests. For example some tests have pre-constructed IP packets with hardcode values and if we change the port they might fail.